apps/designer/app/designer/features/style-panel/shared/flex-grid.tsx

```diff
@@ -0,0 +1,152 @@
+import { Flex, Grid, IconButton } from "@webstudio-is/design-system";
+import { DotFilledIcon } from "@webstudio-is/icons";
+import { useIsFromCurrentBreakpoint } from "./use-is-from-current-breakpoint";
+import type { Style } from "@webstudio-is/react-sdk";
+import type { CreateBatchUpdate } from "./use-style-data";
+
+export const FlexGrid = ({
```

Comment:
this component is in shared folder, looks like it would be shared, but afaik this FlexGrid UI is ever only going to be used once where it is currently used, right? Then why putting it to shared? 

please ask yourself the same question for other things from shared

Reply:
shared should be only ever used when you need to share that functionality, local location is always preferred if no sharing necessary

reason is: co-location principle (high cohesion) + shared should be considered generally more fragile, because used in multiple places, so needs higher quality, more testing etc., so shared is the last resort. 

We can have separate files and still keep it local by creating a folder and an index.ts that only exports public api

Reply:
I would normally be not that serious about a local shared folder inside a feature, since that shared folder is not used outside of the style panel, but I am afraid that style panel is going to be a very large portion of the code base, so we need to be strict about it here